public/js/loginValidation.js

```diff
@@ -0,0 +1,52 @@
+let isValidEmail = (elementID) => {
+    let inputVal = document.getElementById(elementID)
+    let userEmailWarning = document.getElementById('userEmailWarning')
+    let email = inputVal.value.trim();
+    let regEx = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$")
+    if (regEx.test(email)) {
+        userEmailWarning.classList.remove('error')
+        return true
+    } else {
+        userEmailWarning.classList.add('error')
+        return false
+    }
+}
+
+let validateInputs = () => {
+    let cleanedEmailInput = encodeURI(document.getElementById('userEmail').value)
+    let cleanedPasswordInput = encodeURI(document.getElementById('password').value)
+
+    if (isValidEmail('userEmail')) {
+        return {'userEmail' : cleanedEmailInput, 'password': cleanedPasswordInput}
+    } else {
+        document.getElementById('submitWarning').textContent = 'Field input error'
+    }
+}
+
+let sendLoginDetails = async (path, data) => {
```

Comment:
this looks like it has been created to use in a generic manner, however the name is not generic - ties it to only being used for login details

Reply:
changed function name to more generic name.